### DISCUSSION TITLE
8.2.0.4 Beta Release

### DISCUSSION THREAD

**Discussion by MobiFlight-Admin** [MobiFlight/MobiFlight-Connector#409] [Announcements]
This adds:
- [Add YouTube button in Main Menu #399](https://github.com/Mobiflight/MobiFlight-Connector/issues/399)

This fixes:
- [Output config does not retain PWM enabled state #405](https://github.com/Mobiflight/MobiFlight-Connector/issues/405)
- [https://github.com/Mobiflight/MobiFlight-Connector/issues/406](https://github.com/Mobiflight/MobiFlight-Connector/issues/406)
- [Mobiflight Modules settings Cancel button cannot be cancelled #408](https://github.com/Mobiflight/MobiFlight-Connector/issues/408)

<hr /><em>This discussion was created from the release <a href='https://github.com/Mobiflight/MobiFlight-Connector/releases/tag/8.2.0.4'>8.2.0.4 Beta Release</a>.</em>

**Comment by ssewell** [comment]
I confirmed the firmware update for the Pro Micro works as expected now. No issues!
